src/com/codefortomorrow/beginner/chapter2/solutions/VariableTypes.java

```diff
@@ -0,0 +1,34 @@
+package com.codefortomorrow.beginner.chapter2.solutions;
+
+/**
+ * @author ArmeetJatyani
+ * March 2021
+ * 
+ * Define different types of variables
+ */
+
+ public class VariableTypes {
+    public static void main(String[] args) {
+        // write your code here
+        
+        // define an integer variable on line 15
+        int age = 23;
+        
+        // define a float variable on line 18
+        float pi = 3.14f;
+        
+        // define a double variable on line 21
+        double decimal = 23.2352536d;
```

Comment:
If I recall correctly we don't need the `d` at the end when initializing `double`s right? We only need to specify `f` for `float`s.

Reply:
Just doing it for the sake of convention.

Reply:
I'm not sure how common this practice is, but if you want to keep it I recommend a comment that lets them know this isn't required.